### ISSUE TITLE
Flutter 2.0

### ISSUE THREAD

**Issue by momoleu** [flutter-moum/flutter_screenshot_callback#46]
Hello, do you know if we will have an update on this package soon? Thank you for your great work.

**Comment by ousvat** [comment]
You can also use this package from my repo where I did more fixes:
```
screenshot_callback:
    git:
      url: git://github.com/ousvat/flutter_screenshot_callback.git
      ```

**Comment by beygee** [comment]
Fixed 2.0.0.
Sorry for being late.
